src/main/java/org/jenkinsci/plugins/ghprb/upstream/GhprbUpstreamStatusListener.java

```diff
@@ -115,7 +116,7 @@ public void onStarted(AbstractBuild<?, ?> build, TaskListener listener) {
 
     // Sets the status to the build result when the job is done, and then calls the createCommitStatus method to send it to GitHub
     @Override
-    public void onCompleted(AbstractBuild<?, ?> build, TaskListener listener) {
+    public void onCompleted(AbstractBuild<?, ?> build, @Nonnull TaskListener listener) {
```

Comment:
In more detail what is happening is that the parent method from jenkins-core is defined with the `@Nonnull` annotation which would cause a compile warning if a null parameter was sent to this method, because the jenkins team wants it known it's not safe to call `onCompleted()` with a *null* `TaskListener`.

Before this change the `onCompleted` override function in the plugin was stripping off the annotation saying that it was OK to pass it a *null* `TaskListener`.  It's generally not good to strip annotations when inheriting unless there's good reason to, and looking into the implementation of the plugin passing a *null* `TaskListener` to the `onCompleted` function **will ** result in an NPE.

TLDR; It's a safety check to prevent passing of nulls to functions that don't handle them.